package.json

```diff
@@ -36,7 +36,8 @@
     "pretest": "bash bin/update-env.sh",
     "test": "tape 'test/**/*.js' | tap-spec",
     "test-watch": "bash bin/test-watch",
-    "postinstall": "bash bin/postinstall",
+    "postinstall": "node bin/dev-setup.js",
+    "precommit": "npm run prepublish",
```

Comment:
I'm up in the air on whether the Husky dependency is worth it for this hook. On one hand, it is nice IMO to see the git hooks in a declarative fashion in package.json. On the other hand, the hook could still be trivially copied over from the `dev-setup.js` script.